README.md

```diff
@@ -1,3 +1,283 @@
 # LavaQueue
 
-yes, no docs lol.
\ No newline at end of file
+> [!IMPORTANT]
+> This plugin *requires* Lavalink `v4.0.5` or greater
+
+A simple queue plugin for [Lavalink](https://github.com/lavalink-devs/Lavalink) with a REST API.
+
+## Summary
+
+* [Installation](#installation)
+* [API](#api)
+* [Get Queue](#get-queue)
+* [Update Queue](#update-queue)
+* [Next Queue Track](#next-queue-track)
+* [Previous Queue Track](#previous-queue-track)
+* [Add Queue Track](#add-queue-tracks)
+* [Update Queue Track](#update-queue-tracks)
+* [Get Queue Track](#get-queue-track)
+* [Add Queue Track](#add-queue-track)
+* [Delete Queue Track(s)](#delete-queue-track(s))
+* [Move Queue Track](#move-queue-track)
+* [Get Queue History](#get-queue-history)
+* [Get Queue History Track](#get-queue-history-track)
```

Comment:
if something is a sub header it should be indented
the events are missing in here too